docs/development.md

```diff
@@ -11,8 +11,10 @@ that was sent up will be diffed against JupiterOne's understanding of the
 ## The integration framework
 
 This SDK supports building JupiterOne integrations using either JavaScript or
-TypeScript. The execution process expects the integration to produce an object
-conforming to the `IntegrationInvocationConfig` interface:
+TypeScript.
+
+The execution process expects the integration to produce an object conforming to
+the `IntegrationInvocationConfig` interface:
```

Comment:
Yah, I wondered a bit about this myself because I had to maintain this example in two places. I like your suggestion of just linking to the interface and keep the example later on.